source/default/_patterns/00-protons/printing/_elements.scss

```diff
@@ -3,3 +3,14 @@
 // The non-printing variables, functions, and mixins of 00-protons/variables
 // are provided to this file via:
 //   webpack.particle.js: sass-loader.data|includePaths
+
+// Combine all breakpoints into a single string.
+$breakpoints: '';
+@each $bp, $value in $grid-breakpoints {
+  $breakpoints: #{$bp} #{$breakpoints};
+}
+
+// Assign breakpoints to --variable at root for reference by Enquire breakpoints.
+:root {
+  --breakpoints: #{$breakpoints};
```

Comment:
All you need here is:

```scss
:root {
  --breakpoints: #{map_keys($grid-breakpoints)};
}
```

This gives a comma+space delimited string of breakpoints, no need for `@each`.

Reply:
@Illepic, I like this and feel a bit foolish for never using it before. Added in the latest push.